test/e2e/network/netpol/kubemanager.go

```diff
@@ -63,12 +63,20 @@ func (k *kubeManager) initializeCluster(model *Model) error {
 		for _, pod := range ns.Pods {
 			framework.Logf("creating/updating pod %s/%s", ns.Name, pod.Name)
 
-			kubePod, err := k.createPod(pod.KubePod())
+			thePod := pod.KubePod()
+			if framework.NodeOSDistroIs("windows") {
+				thePod.Spec.NodeSelector = map[string]string{
+					"kubernetes.io/os": "windows",
+				}
+			} else {
+				framework.Logf("node distro is NOT WINDOWS !!!!!!!!!!!")
```

Comment:
is this really needed or was just for debugging?
we already have too many logs, and this doesn't give much value, I assume people will know which cluster they are using

Reply:
oops :) fixing